web/src/app/app-service/app-service-list/app-service-list.component.html

```diff
@@ -21,11 +21,11 @@
          <i class="fa fa-list mr-2 text-danger"></i>
          <span class="font-weight-bold">App Service List</span>
      </div>
-     <div class="card-header py-2">
+     <!-- <div class="card-header py-2">
          <a class="btn btn-sm btn-primary" routerLink="../add-app-service">
              <i class="fa fa-plus mr-2"></i>New App
          </a>
-     </div>
+     </div> -->
```

Comment:
Why not just remove it?

Reply:
I'll just comment it out for the possibility of using it again